ga4gh/client.py

```diff
@@ -46,17 +61,14 @@ def _usingWorkaroundsFor(self, workaround):
         return workaround in self._workarounds
 
     def _debugResponse(self, jsonString):
-        if self._debugLevel > 1:
-            # TODO use a logging output and integrate with HTTP client more
-            # nicely... also, logging / error handling in general
-            print("json response:")
-            pp = json.dumps(
-                json.loads(jsonString), sort_keys=True, indent=4)
-            print(pp)
+        self.logger.debug("json response:")
+        pp = json.dumps(
```

Comment:
The trouble with this is we're doing a lot of JSON manipulation even when _debugLevel = 0, which would probably slow things down a lot. Maybe we should remove the JSON formatting? Is there a clean way to keep pretty printing (which is handy) but without having to construct the string every time?